src/basics/basics.md

```diff
@@ -1,8 +1,6 @@
 # Basics
 
-Now that we've successfully compiled a program and executed it within the context of our Arch VM, let's dig into the details of how an Arch [program] is written and discuss some best practices.
-
-In this section, we'll guide you through the basics of writing functions and handling input parameters effectively within a demo program: [GraffitiWall].
+To illustrates the basic process involved in developing an Arch [program] we're going to build, deploy and interact with a demo program: [GraffitiWall].
```

Comment:
This English doesn't make sense. I think it's a typo or something's missing.

Reply:
My bad, It's the `s` in `illustrates`

Reply:
https://github.com/Perelyn-Arch/arch-book/commit/b07ef43cc2f7993126d352371ee9da2c35d33946